cassandra-3.11/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM openjdk:8u292-slim
 
-ARG STARGATE_VERSION=v1.0.50
+ARG STARGATE_VERSION=1.0.51
```

Comment:
this seems wrong `v` is missing

Reply:
Thank you @ivansenic fixed; added a note in Release Instructions as well.